src/core/oplist

```diff
@@ -800,6 +800,8 @@ unicmp_s            w(int64) r(str) r(str) r(int64) r(int64) r(int64) :pure
 setdispatcherfor    r(obj) r(obj)
 getstrfromname       w(str) r(str) :pure
 indexic_s            w(int64) r(str) r(str) r(int64) :pure
+get_port_sk          w(int64) r(obj)
```

Comment:
We don't put `_` inside opnames except (a) a prefix to specify they are not normal ops (for example `sp_` for specialized ops) or a suffix to indicate type (for example `_i`). So this should be renamed to `getport_sk`.